vprdb/converters/kitti_converter.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import mrob
+import numpy as np
+import open3d as o3d
+import pykitti
+import shutil
+
+from pathlib import Path
+
+
+def copy_img_folder(src_folder, out_folder, shape):
```

Comment:
Shape?
I also suggest adding types info in this function and the function below
```suggestion
def copy_img_folder(src_folder: Path, out_folder: Path, dataset_len: int):
```